### DISCUSSION TITLE
Ctrlr on Android

### DISCUSSION THREAD

**Discussion by hungmanmoojoo** [RomanKubiak/ctrlr#315] [General]
Hi. i'm sorry if this seems like a stupid question for you but i'm not that well versed in what an Android device can and cannot do so;

what is needed to get Ctrlr running on an Android with a working panel?

**Comment by Godlike-Productions** [comment]
You'd have to compile it from source for that target.  You would probably have to add the Android target in Producer and add functionality and coding for Android from there.  I have no idea what would be involved.
